### PR TITLE
OSDOCS-10548 Adding network minimum requirements for ROSA Classic

### PR DESCRIPTION
**Pull request by laubai**
Version(s): 4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-10548

Link to docs preview:
- [Prerequisites checklist](https://78475--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_planning/rosa-cloud-expert-prereq-checklist.html#mos-network-prereqs-min-bandwidth_rosa-cloud-expert-prereq-checklist)
- [Detailed requirements for STS](https://78475--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-aws-prereqs.html#rosa-network-prereqs_rosa-sts-aws-prereqs)
- [Detailed requirements for non-STS](https://78475--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs#rosa-without-sts-network-prereqs_prerequisites)

QE review:
- [x] QE has approved this change.

Additional information:
Approving SMEs are not available until next week so pushing to peer review first, and will hold off on MR until SME and QE review are complete.

### CODE REVIEW

**Review by tmalove — APPROVED** [review]

**Review by tmalove — APPROVED** [review]

File: modules/mos-network-prereqs-min-bandwidth.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ifdef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-osd,openshift-azure[]
+:mos-min-bandwidth: 120{nbsp}Mbps
+endif::[]
+
+[id="mos-network-prereqs-min-bandwidth_{context}"]
+= Minimum bandwidth
+
+During cluster deployment, {product-title} requires a minimum bandwidth of {mos-min-bandwidth}. On networks slower than {mos-min-bandwidth}, the deployment process times out and deployment fails.
+
+After deployment, network requirements are determined by your workload. However, Red Hat recommends a minimum bandwidth of {mos-min-bandwidth} to ensure timely cluster and operator upgrades.
```
[the remark this answers is not in the document]
Comment (laubai) [reply modules/mos-network-prereqs-min-bandwidth.adoc:15]: Changed to "However, a minimum bandwidth of {mos-min-bandwidth} helps to ensure timely cluster and operator upgrades." Thanks Tami!

**Pull request merged**